instrumentation/cassandra-datastax-3.8.0/src/test/java/com/nr/agent/instrumentation/cassandra/CassandraTest.java

```diff
@@ -49,9 +50,14 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
+/*
+ * Gradle hangs indefinitely when these tests run on Java 16
+ *
+ * FIXME capture jmap/jstack thread dumps?
```

Comment:
Is this FIXME meant to stay for now?

Reply:
I'm ok with removing or leaving it. It's mostly there to spark inspiration in some poor future soul who decides to actually investigate the issue.